tests/test_order.py

```diff
@@ -134,12 +135,14 @@ def test_place_order_no_id(order_parameters, auth_mock, order_mock, requests_moc
         Order.place(auth_mock, order_parameters)
 
 
-@pytest.mark.skip(reason="Placing orders costs credits.")
+#@pytest.mark.skip(reason="Placing orders costs credits.")
```

Comment:
ques: why do we want to comment this?
I am guessing since this is marked as a live test, this will be skipped anyway (unless specified to run live tests)?

Reply:
I forgot to uncomment :(